pkg/runner/dochandlers/v22/v22.go

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: Apache-2.0
+package v22
```

Comment:
@nishakm Effective Go recommends naming packages without underscores.
I'm not sure how can we differentiate between a major and a minor version apart from this

https://go.dev/doc/effective_go#package-names:~:text=By%20convention%2C%20packages%20are%20given%20lower%20case%2C%20single%2Dword%20names%3B%20there%20should%20be%20no%20need%20for%20underscores%20or%20mixedCaps